src/main/java/uk/gov/hmcts/dts/fact/controllers/admin/AdminRole.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.hmcts.dts.fact.controllers.admin;
+
+final class AdminRole {
```

Comment:
Really. for my own interest and for thought, can you use @NoArgsConstructor here in order to replace

private AdminRole() {
    } ?

Reply:
Not sure I will need to try. This private constructor was added to address PMD complaint